galaxy/values.yaml

```diff
@@ -132,6 +132,24 @@ extraVolumeMounts: []
 
 initJob:
   createDatabase: true
+  externalConfs:
```

Comment:
I feel like this should be renamed to something like 'downloadToolConfs' or similar, as it could be confused with configuration in general?